code/modules/mob/living/carbon/human/species/unathi.dm

```diff
@@ -102,7 +102,18 @@
 				to_chat(user, "<span class='warning'>You run out of momentum!</span>")
 				return
 
-
-
 /datum/species/unathi/handle_death(mob/living/carbon/human/H)
 	H.stop_tail_wagging(1)
+
+/datum/species/unathi/ashwalker
```

Comment:
We probably want to make them a bit more unique. Anyone up for porting over the digitigrade stuff from /tg/? The blurb also needs a rewrite.

Reply:
We could techncially do the icon stuff for it since tiger updated the icon system a while back. @Markolie do you just need the sprites or what?

Reply:
I’m completely unfamiliar with that system, so I’d prefer someone else doing it. That along with the Angel species and perhaps Pod People. /tg/‘s species icon system is a bit too different from ours.

Reply:
I’m completely unfamiliar with that system, so I’d prefer someone else doing it. That along with the Angel species and perhaps Pod People. /tg/‘s species icon system is a bit too different from ours.